activerecord/test/models/company.rb

```diff
@@ -10,7 +10,6 @@ class Company < AbstractCompany
   has_one :dummy_account, :foreign_key => "firm_id", :class_name => "Account"
   has_many :contracts
   has_many :developers, :through => :contracts
-  has_many :accounts
```

Comment:
Turns out that definition originated in this commit: 99307b959b1c16ab1dd0400a7398019fe9d5494b - where I don't think it was actually used. :smile: